recipes/asyncpg/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "asyncpg" %}
+{% set version = "0.12.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "9bf6588d9e9dd0fe98b9df7d7e46680c4e9c25e55b12b49bc3f23924e3e53d7d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  # Not using pypi URL because the source dist has incorrect C files for windows
+  url: https://github.com/MagicStack/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Might be worth raising this issue upstream.

Reply:
FWIW there seems to be a [build option]( https://github.com/MagicStack/asyncpg/blob/v0.12.0/setup.py#L33-L34 ) to build with Cython directly (even when the C files are present).